services/webdav/pkg/service/v0/search.go

```diff
@@ -161,7 +143,7 @@ func matchToPropResponse(ctx context.Context, match *searchmsg.Match) (*propfind
 		return nil, err
 	}
 	response := propfind.ResponseXML{
-		Href:     net.EncodePath(path.Join("/remote.php/dav/spaces/", ref)),
+		Href:     net.EncodePath(path.Join("/dav/spaces/", ref)),
```

Comment:
Could you please double check whether this change is correct? Or should we try to make this dynamic depending on whether the `remote.php` has been part of the request endpoint?